index.html

```diff
@@ -69,69 +69,23 @@ <h2 class="roboto-font">I’am a Software Developer</h2>
+            <div id="line-works"> 
+                <hr>
+            </div>  
         </header>
-        <section class="d-flex" id="first-card">
-            <img src="images/worksImage.png" alt="Yoga Page">
-            <h2 class="heading-font">Multi-Post Stories</h2>
-            <p class="inter-font">A daily selection of privately personalized reads; 
-                no accounts or sign-ups required. has been 
-                the industry's standard dummy text ever since 
-                the 1500s, when an unknown printer took a standard dummy text.</p>
-            <ul class="used-languages d-flex" id="languages">
-                <li>
-                    <div class="inter-font">css</div>
-                </li>
-                <li>
-                    <div class="inter-font">html</div>
-                </li>
-                <li>
-                    <div class="inter-font">bootstrap</div>
-                </li>
-                <li> 
-                    <div class="inter-font">Ruby</div>
-                </li>
-            </ul>
-            <button class="inter-font see-btn" id="first-btn"  type="button"> See Project </button>
-        </section>
-    </section>
-
-    <section  class="d-flex-center" id="work-cards">
-        <section class="work-card">
-            <section class="card-container">
-                <header>
-                    <h2 class="heading-font">Profesional Art Printing Data</h2>
-                </header>
-                <p  class="inter-font">
-                    A daily selection of privately personalized reads;
-                    no accounts or sign-ups required. 
-                    has been the industry's standard
-                </p>
-                <ul class="used-languages d-flex card-languages">
-                    <li>
-                        <div class="inter-font">html</div>
-                    </li>
+        <section class="d-flex" id="first-card">  
+            <img src="images/worksImage.svg" alt="Yoga Page" id="card-image">
+            <div id="info-container">
+                <h2 class="heading-font">Multi-Post Stories</h2>
+                <p class="inter-font">A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. has been 
+                    the industry's standard dummy text ever since 
+                    the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="d-flex" id="languages">
                     <li>
-                        <div class="inter-font">bootstrap</div>
+                        <div class="inter-font">css</div>
                     </li>
-                    <li> 
-                        <div class="inter-font">Ruby</div>
-                    </li>
-                </ul>
-            </section>
-            <button class="inter-font see-btn d-flex-center card-btn"  type="button">See Project</button>
-        </section>
```

Comment:
- [ ] You have done a great job removing the overlay black color on these cards on hover, there is one more thing you will need to fix, let's make the texts and technologies on the card disappear on hover as well to have the same thing as in the Figma design. This is what should happen when we hover on a work card 👇 , as you can see the texts have disappeared.
![image](https://user-images.githubusercontent.com/55465133/130147842-fe9cd29b-22dc-4cde-b5bf-4e6f753b572d.png)

- Hint: You can do something similar to this 👇 
```
.work-card:hover > .card-container > header,
.work-card:hover > .card-container > .inter-font, 
.work-card:hover > .card-container > .used-languages {
 display: none;
}

```
This will make all the texts disappear when we hover over the card, then you can do a similar approach to show the `See project` button when all the texts and used-languages are hidden, I am sure you can do this  😉